src/app/lib/config/services/afaanoromoo.ts

```diff
@@ -140,7 +140,7 @@ export const service: DefaultServiceConfig = {
           },
           accept: 'TOLE',
           reject: 'Maaltu akka jijjirame bari',
-          rejectUrl: 'https://www.bbc.com/usingthebbc/your-data-matters',
+          rejectUrl: 'https://www.bbc.co.uk/usingthebbc/privacy-policy/',
```

Comment:
This and all other 40 WS services (apart from news/sport etc) should use .com 
```suggestion
          rejectUrl: 'https://www.bbc.com/usingthebbc/privacy-policy/',
```

Reply:
Good spot, thanks - will change